pages/common/bedops.md

```diff
@@ -0,0 +1,24 @@
+# bedops
+
+> BEDOPS: A fast, highly scalable, and easily-parallelizable genome analysis toolkit.
+> Apply efficient set operations on any number of sorted BED or Starch files such as element-of, intersection, merge, difference, partition, etc. and their logical inverses.
+
+- Take the multiset union of two or more BED or Starch files and write the sorted result to {{path/to/output_file}}:
+
+`bedops --everything {{path/to/file_1}} {{path/to/file_2}} [{{path/to/file_3}} ...] > {{path/to/output_file}}`
+
+- Find elements from reference set {{path/to/reference_file}} that overlap one or more query sets by one or more bases:
+
+`bedops --element-of 1 {{path/to/reference_file}} {{path/to/query_file_1}} [{{path/to/query_file_2}} ...] > {{path/to/output_file}}`
+
+- Merge the overlapping intervals in one or more files and write the sorted result to {{path/to/output_file}}:
+
+`bedops --merge {{path/to/file_1}} [{{path/to/file_2}} {{path/to/file_3}} ...] > {{path/to/output_file}}`
+
+- Partition overlapping intervals in one or more files into disjoint intervals and write the sorted result to {{path/to/output_file}}:
+
+`bedops --partition {{path/to/file_1}} [{{path/to/file_2}} {{path/to/file_3}} ...] > {{path/to/output_file}}`
+
+- For all operations, efficiently apply set operations on specified {{chromosome}} and ignore other chromosomes:
```

Comment:
I think the {{}} can be removed from the descriptions. They are generally used in the examples. I suggest ".. efficiently apply set operations on a specified chromosome and ignore .. "

Reply:
I think I just reviewed how other binaries were explained in their own Markdown files, and I saw this convention used there. I'm happy to do things in whichever way you would prefer.

Reply:
If this convention is used in other files, that is incorrect and needs to be removed. Sometimes, these things skip reviews and get committed to the repo.